.github/workflows/in_develop_labeler.yml

```diff
@@ -8,7 +8,7 @@ jobs:
   label_preflight:
     name: Label Issue In Develop
     runs-on: ubuntu-latest
-    if: github.event.pull_request.merged == true
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
```

Comment:
Nice, this is much better! 
Didn't know of the `.conclusion` event.

We should probably have a clause for failure as well, maybe we can just console log something along the lines of 
> Label Preflight workflow failed, please check the logs [Link to preflight workflow run]

So maybe something like this?

```yml
   on-failure:
     runs-on: ubuntu-latest
     if: ${{ github.event.workflow_run.conclusion == 'failure' }}
     steps:
       - run: echo "First workflow was a failure"
 ```